Solutions/IoTOTThreatMonitoringwithDefenderforIoT/Analytic Rules/IoTDenialofService.yaml

```diff
@@ -13,12 +13,14 @@ queryPeriod: 5m
 triggerOperator: gt
 triggerThreshold: 0
 tactics:
-  - Impact
+  - InhibitResponseFunction
+relevantTechniques:
+  - T0814
 query: |
   SecurityAlert
-  | where ProductName == "Azure Security Center for IoT"
+  | where ProviderName == "IoTSecurity"
   | where AlertName == "Suspicion of Denial Of Service Attack"
-  | where TimeGenerated <= ProcessingEndTime + 60m
+  | where tostring(parse_json(ExtendedProperties).isNew) == "True"
   | extend DeviceId = tostring(parse_json(ExtendedProperties).DeviceId)
```

Comment:
we can also use single piped extend with columns seperated by ','

Reply:
done @vakohl